discovery-provider/plugins/notifications/src/__tests__/mappings/usdcPurchaseBuyer.test.ts

```diff
@@ -17,9 +17,9 @@ import {
 
 import { AppEmailNotification } from '../../types/notifications'
 import { renderEmail } from '../../email/notifications/renderEmail'
-import { RepostType } from '../../types/dn'
+import { usdc_purchase_content_type } from '../../types/dn'
 
-describe('Repost Notification', () => {
+describe('USDC Purchase Buyer', () => {
```

Comment:
is this diff messing with me? or was it committed as repost before for some reason?